integration-tests/basic-test-app/ios/Podfile.lock

```diff
@@ -1500,9 +1500,28 @@ PODS:
     - React-logger (= 0.74.1)
     - React-perflogger (= 0.74.1)
     - React-utils (= 0.74.1)
-  - RNEmbrace (5.0.2):
+  - RNEmbraceCore (5.0.2):
```

Comment:
hmm I'm curious on this change, it looks like previously the dependencies of this pod was just:
```
  - RNEmbrace (5.0.2):
    - EmbraceIO (= 6.5.1)
    - React-Core
```

might be expected but do you know what changes to the package caused all the other ones to be pulled in? (FWIW seems like TracerProvider, etc. also have the same larger list of dependencies)

Reply:
oh I guess it's due to this in the podspec:
```

  # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
  # See https://github.com/facebook/react-native/blob/febf6b7f33fdb4904669f99d795eba4c0f95d7bf/scripts/cocoapods/new_architecture.rb#L79.
  if respond_to?(:install_modules_dependencies, true)
    install_modules_dependencies(s)
  else
    s.dependency "React-Core"
```

which seems expected and the way RN wants to do things on newer versions?

Reply:
yes, it's expected. this is because I updated the previous .podspec file to use the same structure + dependencies as what `create-react-native-library` creates. I don't see a reason why we don't update this podspec to be in sync with the other packages